src/main/java/org/fisco/bcos/web3j/precompile/common/PrecompiledCommon.java

```diff
@@ -64,13 +65,30 @@ public static String transferToJson(int code) throws JsonProcessingException {
   public static String getJsonStr(String output) throws JsonProcessingException {
     try {
       int code = 0;
-      if ("f".equals(output.substring(2, 3))) {
-        code = -1;
-      } else {
-        code = new BigInteger(output.substring(2, output.length()), 16).intValue() - 256;
-        if (code < -200) {
-          code = Integer.valueOf(output.substring(2), 16).intValue();
-        }
+      code = new BigInteger(output.substring(2, output.length()), 16).intValue();
+      if(code == 1)
+      {
```

Comment:
Could you add a comment or TODO to explain why this error code translation is necessary now? What will we refact this part in the future?